udata/core/discussions/permissions.py

```diff
@@ -4,16 +4,27 @@
     OrganizationEditorNeed,
 )
 
+from .models import Message
+
 
 class CloseDiscussionPermission(Permission):
     def __init__(self, discussion):
         needs = []
         subject = discussion.subject
 
-        if getattr(subject, "organization"):
+        if getattr(subject, "organization", None):
             needs.append(OrganizationAdminNeed(subject.organization.id))
             needs.append(OrganizationEditorNeed(subject.organization.id))
         elif subject.owner:
             needs.append(UserNeed(subject.owner.fs_uniquifier))
 
         super(CloseDiscussionPermission, self).__init__(*needs)
+
+
+class DiscussionMessagePermission(Permission):
+    def __init__(self, message: Message):
+        needs = []
+
+        needs.append(UserNeed(message.posted_by.fs_uniquifier))
```

Comment:
I don't think this is intuitive that only the `posted_by` user can edit, if there is a `posted_by_organization`